src/vs/platform/extensionManagement/common/extensionTipsService.ts

```diff
@@ -368,11 +374,17 @@ export abstract class AbstractNativeExtensionTipsService extends ExtensionTipsSe
 						.replace('%ProgramFiles%', () => env['ProgramFiles']!)
 						.replace('%APPDATA%', () => env['APPDATA']!)
 						.replace('%WINDIR%', () => env['WINDIR']!));
+					// TODO: Should we look for <LOCALAPPDATA>?
+					// .replace('%LOCALAPPDATA%', () => env['LOCALAPPDATA']!)
+					// Example we are missing out on: <C:\Users\myName\AppData\Local\Programs>
 				}
 			} else {
 				exePaths.push(join('/usr/local/bin', exeName));
 				exePaths.push(join('/usr/bin', exeName));
 				exePaths.push(join(this.userHome.fsPath, exeName));
+				// TODO: Should we look for </opt/homebrew/bin>?
```

Comment:
Could not test this since `executableTips` were always empty for me
Same story with LOCALAPPDATA

Reply:
You have to copy `exeBasedExtensionTips` from product.json into `product.overrides.json` to get these populated